documentation/RLC-quickstart.md

```diff
@@ -1,71 +1,96 @@
-Getting Started - Generate the RLC rest-level client libraries with TypeSpec
-========================================================================
+Getting Started: Generate JavaScript/TypeScript RLC Libraries with TypeSpec
+===========================================================================
 
 # Before you start
 
-Please refer to this [link](https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md#prerequisites) for the environment set up prerequisites in azure-sdk-for-js repository. We highly recommand to read [this blog](https://devblogs.microsoft.com/azure-sdk/azure-rest-libraries-for-javascript/) to get familiar with REST libraries for JavaScript. 
+:warning: Ensure that your TypeSpec definition has been merged into the main branch of the [Azure REST API specs repository](https://github.com/Azure/azure-rest-api-specs) before you begin.
 
 :warning: Note: if you’re still generating from Swagger with RLC, please read [this doc](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/RLC-Swagger-quickstart.md) for Swagger specific details.
```

Comment:
maybe we should remove this, I don't want to sound like Swagger is still an option. I would rather have them ask us about swagger and get guidance to move to TypeSpec than having them go all the way to generate from swagger and have to go back